src/emit.iced

```diff
@@ -122,6 +122,15 @@ exports.GoEmitter = class GoEmitter
     @untab()
     @output ")"
 
+    @output "var #{t.name}_map = map[string]#{t.name}{"
```

Comment:
Ok, the values are like `ProofStatus_NONE`, which is better `NameMap` or `Name_Map`?